portality/api/v2/data_objects/common.py

```diff
@@ -0,0 +1,16 @@
+def _check_for_script(data):
+    print(data)
```

Comment:
I don't think we want print statements in the low level API

Reply:
Nope. This is overlooked from development.